deploy/olm-catalog/3scale-operator-master/0.0.0/3scale-operator-master.v0.0.0.clusterserviceversion.yaml

```diff
@@ -460,7 +460,7 @@ spec:
               labels:
                 com.redhat.component-name: 3scale-operator
                 com.redhat.component-type: infrastructure
-                com.redhat.component-version: 0.6.0
+                com.redhat.component-version: 0.7.0
```

Comment:
Wouldn't this be 0.6.1? Release is 3scale 2.9.1 (and OLM CSVs of the 2.9.x channel are 0.6.x)

Reply:
Sorry, I got confused, this is to merge on master for 2.10. So this is ok :+1: